ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -262,6 +262,7 @@ class Metrics(val registry: MetricRegistry) {
           registry.timer(prefix :+ "lookup_ledger_configuration")
         val lookupKey: Timer = registry.timer(prefix :+ "lookup_key")
         val lookupActiveContract: Timer = registry.timer(prefix :+ "lookup_active_contract")
+        val lookupActiveContractValueTranslation: Timer = registry.timer(prefix :+ "lookup_active_contract_value_translation")
```

Comment:
Probably worth adding one level of nesting here for things that are not actually on the database.